packages/postgresql/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.32.0"
+  changes:
+    - description: Add Alerting Rule Templates.
```

```suggestion
    - description: Add alerting rule templates.
```

Uniform CL entries across all PRs.